### DISCUSSION TITLE
Why is error.vue in the layouts folder?

### DISCUSSION THREAD

**Discussion by btkostner** [nuxt/nuxt#9538] [Questions (Nuxt 2)] [answered]
I understand that `error.vue` is in `layouts/` and not `pages/`, but I'm wondering about the technical decision for that. What prevents nuxt from changing it to something like `pages/_error.vue` as a special page? I think it would make more sense for it to be there as it works more like a page than a layout, plus it (might) allow for some cool features like different error pages for different nested folders (`pages/_error.vue` and `pages/blog/_error.vue`.)

**Comment by TheAlexLichter** [comment, marked as answer]
Mostly to avoid breaking changes. This will be adjusted in Nuxt 3 (using `pages/_error.vue` then).

The idea of nested error pages sounds interesting!

**Reply by btkostner** [reply]
Awesome, I didn't realize it was changed in Nuxt 3. Thanks for the fast answer!

**Reply by agnelnieves** [reply]
Nice to know!
